locales/es/LC_MESSAGES/hidden-curriculum.po

```diff
@@ -132,561 +168,665 @@ msgid ""
 "home means grants are perceived as risky, and 2) Aleksey is still "
 "learning how to engage fully in open science practices."
 msgstr ""
+"Aleksey tiene poco más de 50 años, nació y creció en Ucrania, donde también "
+"Actualmente vive y trabaja. Es genetista de formación, trabaja con "
+"moscas de la fruta. Aleksey está a mitad de su carrera y trabaja como investigador asociado "
+"Científico (un puesto permanente y permanente) en Kiev/Kiev. Su objetivo es "
+"ampliarse hacia nuevos métodos experimentales dentro de una investigación de larga duración"
+"programar y recibir subvenciones para dedicarse a la investigación (no pueden recibir "
+"salario de subvenciones). La financiación pagaría un asistente de investigación de doctorado o "
+"posdoctorado, así como pagar recursos para realizar actividades de investigación".
+"Sus intereses profesionales incluyen la genética molecular y la biología celular",
+"Enfermedades raras, con modelado computacional (pero no con código abierto ".
+"software). Aleksey ha recibido subvenciones locales antes pero fracasó en "
+"obtener financiación de un financiador internacional, como era su propuesta"
+"mal calibrado para la solicitud). Aleksey es una persona con discapacidad ",
+"necesita tecnología de asistencia de voz y también es padre de dos hijos"
+"adolescentes. Los desafíos relacionados con la obtención de una beca internacional "
+"incluir idioma (la lengua nativa de Aleksey, el ucraniano, tiene cirílico"
+"alfabeto), que presenta una barrera para enviar en inglés y "
+"colaborar internacionalmente. Desafíos para postularse con éxito "
+"La financiación de subvenciones incluye: 1) la inestabilidad geopolítica actual en Aleksey"
+"En casa significa que las subvenciones se perciben como riesgosas, y 2) Aleksey todavía lo es "
+"aprender a participar plenamente en prácticas científicas abiertas".
```

Comment:
```suggestion
"vive y trabaja actualmente. Es genetista de formación y trabaja con "
"moscas de la fruta. Aleksey está a mitad de su carrera professional y trabaja como Investigadore Asociade "
"Científique (un puesto de titular permanente) en Kiev/Kyiv. Su objetivo es "
"abrir horizontes hacia nuevos métodos experimentales dentro de un programa de investigación "
"a largo plazo y recibir subvenciones para dedicar a la investigación (elle no puede recibir "
"salario de becas). La financiación pagaría un doctorande o "
"post-doctorade, así como pagar recursos para realizar actividades de investigación. "
"Sus intereses profesionales incluyen la genética molecular y la biología celular, "
"enfermedades raras, con modelado computacional (pero no con código abierto). "
"Aleksey ha recibido subvenciones locales antes pero fracasó intentando "
"obtener fondos de un financiador internacional, ya que su propuesta "
"no se adecuaba a la solicitud). Aleksey es una persona con discapacidad, "
"necesita tecnología de asistencia de voz y también es padre de dos "
"adolescentes. Los desafíos relacionados con la obtención de una beca internacional "
"incluyen el idioma (su lengua materna, el ucraniano, usa el alfabeto cirílico), "
"lo que dificulta enviar una solicitud en inglés y "
"colaborar internacionalmente. Los desafíos al conseguir financiación internacional "
"incluyen: 1) la inestabilidad geopolítica actual en en su región "
"hace que las subvenciones se perciban como algo arriesgado, y 2) Aleksey todavía está "
"aprendiendo a participar plenamente en prácticas científicas abiertas".
```

There are some parts of this I am not very confident about, spanish isn't gender neutral so people have recently started trying to intorduce gender neutral things like `elle` instead of él (he) or ella (she). I am mostly copying from Catalan gender neutral.

Reply:
Hi @OriolAbril I get the confusion around the gender-neutral terms in Spanish as it is still evolving. I am fine with whatever feels clearer, If you think it’s better to stick with "investigador/a" and "doctorando/a" while keeping the rest of your changes, that also works well for me.
Let me know what you think.